src/app/routes/frontend.server.routes.js

```diff
@@ -14,7 +14,9 @@ module.exports = function (app) {
     celebrate({
       [Segments.QUERY]: {
         redirectPath: Joi.string()
-          .regex(/^[a-fA-F0-9]{24}(\/(preview|template|use-template))?$/)
+          .regex(
+            /^[a-fA-F0-9]{24}(\/(preview|template|use-template))?(\?([0-9A-Za-z]+=[0-9A-Za-z]+)(&[0-9A-Za-z]+=[0-9A-Za-z]+)*)?$/,
```

Comment:
ideally, there would be a way to rewrite this so that we don't have to deal with a monster regex

Reply:
as above, i've relaxed the validation for this